clippy.toml

```
@@ -0,0 +1,2 @@
+# Use fxhash::{FxHashMap, FxHashSet} whenever possible
+disallowed-types = ["std::collections::HashMap", "std::collections::HashSet"]
```

You can add reasons that clippy will print:
```suggestion
disallowed-types = [
    { path = "std::collections::HashMap", reason = "use FxHashMap for performance reasons" },
    { path = "std::collections::HashSet", reason = "use FxHashSet for performance reasons" }, 
]
```

---

I'm need extracción the code is more problems that solution